acceptancetests/jujupy/k8s_provider/gke.py

```diff
@@ -204,10 +202,10 @@ def log_remaining(remaining, msg=''):
         logger.info('waiting for cluster fully provisioned.')
         for remaining in until_timeout(600):
             try:
-                cluster = self._get_cluster(self.gke_cluster_name)
+                cluster = self._get_cluster(self.cluster_name)
                 if cluster.status == CLUSTER_STATUS.RUNNING:
                     return
-            except Exception as e: # noqa
+            except Exception as e:  # noqa
```

Comment:
remove extra space?

Reply:
it should be 2 * " " for inline comment
https://www.flake8rules.com/rules/E262.html